plugins/language_json.lua

```diff
@@ -0,0 +1,28 @@
+-- mod-version:3
```

Comment:
```suggestion
-- mod-version:3 priority:90
```

Since language_js.lua already supports .json files the priority of this plugin should be higher than default so it gets loaded and applied before language_js.

Reply:
Didn't knew that was possible, cheers!

Maybe the `.json` file extension should be removed from `language_js` aswell, or should it be kept as backup?

Reply:
I guess it should be kept to provide some json support on default installs but the maintainers will tell.

Reply:
Coming  back to this after doing some testing, it seems like `language_js` is loaded before `language_json`, but after removing `priority:90` it works fine again